superbuild/ConfigureExternalProjects.cmake

```diff
@@ -154,6 +154,12 @@ else()
   set(GITLAB_INRIA_PREFIX https://gitlab.inria.fr/)
 endif()
 
+if(${USE_GITLAB_SSH})
```

Comment:
this was not done automatically?

Reply:
Not on my side

Reply:
Interesting bug, we need to check if the bug is there also on medInria. And since i had some problem compiling medInria with USE_GITLAB_SSH, it could be